simplq/src/components/pages/Admin/ControlScroll.js

```diff
@@ -1,25 +1,37 @@
 // left: 37, up: 38, right: 39, down: 40,
 // spacebar: 32, pageup: 33, pagedown: 34, end: 35, home: 36
-let keys = {37: 1, 38: 1, 39: 1, 40: 1};
+const keys = { 37: 1, 38: 1, 39: 1, 40: 1 };
 
-const preventDefault = (e) => e.preventDefault();
+function preventDefault(e) {
+  e.preventDefault();
```

Comment:
Is there any specific reason why these were changed?

Reply:
Arrow functions were giving lint issues. They should always return a value for eslint to be kept happy. That was cluttering the function with unwanted `return undefined;` which was looking confusing from readability point.

So then standardised to use `function` as other two functions in the module are already functions.

Reply:
Okay